source/documents.txt

```diff
@@ -45,7 +45,7 @@ tab loads, Compass displays all documents that match your query criteria
 (or all documents within the sample if no query criteria was
 specified).
 
-.. figure:: /images/compass/doc-view.svg
+.. figure:: /images/compass/doc-view.png
   :figwidth: 800px
```

Comment:
This will stretch the image in an odd fashion. You should keep it to its exact size: `810px`. I would check the rest accordingly.